torch/nn/modules/activation.py

```diff
@@ -699,7 +699,7 @@ class Hardshrink(Module):
         \end{cases}
 
     Args:
-        lambd: the :math:`\lambda` value for the Hardshrink formulation. Default: 0.5
+        lambda: the :math:`\lambda` value for the Hardshrink formulation. Default: 0.5
```

Comment:
I think the arg name is actually `lambd` (to avoid conflicting with the Python keyword `lambda`):
https://github.com/pytorch/pytorch/blob/9444d38c77790e2a47e0c21b6e9a65142e44d2d4/torch/nn/modules/activation.py#L719-L721